components/ArtPiecePreview/ArtPiecePreview.js

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function ArtPiecePreview({ artPiece }) {
```

Comment:
The user story tells us to take the props `image`, `title` and `artist` here. You could adjust this here and below in the rendered JSX :)

Reply:
Thank you! Updated the props ✅